### PR TITLE
[15.0][MIG] sale_stock_last_date: Migration to version 15.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
CC @Tecnativa TT37316

@CarlosRoca13 @sergio-teruel Please review.

### CODE REVIEW

File: sale_stock_last_date/views/sale_order_view.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+                expr="//field[@name='order_line']//form//field[@name='customer_lead']"
+                position="after"
+            >
+                <field name="last_delivery_date" />
+            </xpath>
+            <xpath
+                expr="//field[@name='order_line']/tree/field[@name='state']"
+                position="before"
+            >
+                <field name="last_delivery_date" />
```
Comment (CarlosRoca13) [inline sale_stock_last_date/views/sale_order_view.xml]:
```suggestion
                <field name="last_delivery_date" optional="show"/>
```

**Review by CarlosRoca13 — APPROVED** [review]
Just a minor comment

**Comment by stefan-tecnativa** [comment]
Change done. Thank you!

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**